sdk/servicebus/Azure.Messaging.ServiceBus/tests/Infrastructure/ServiceBusTestEnvironment.cs

```diff
@@ -17,35 +17,23 @@ namespace Azure.Messaging.ServiceBus.Tests
     ///
     public class ServiceBusTestEnvironment: TestEnvironment
     {
-        public ServiceBusTestEnvironment() : base("servicebus")
-        {
-            ActiveServiceBusNamespace = new Lazy<NamespaceProperties>(EnsureServiceBusNamespace, LazyThreadSafetyMode.ExecutionAndPublication);
-            ParsedConnectionString = new Lazy<ConnectionStringProperties>(() => ConnectionStringParser.Parse(ServiceBusConnectionString), LazyThreadSafetyMode.ExecutionAndPublication);
-        }
+        /// <summary>The name of the environment variable used to specify the Service Bus namespace to use for the test run.</summary>
+        public const string ServiceBusConnectionStringEnvironmentVariable  = "SERVICEBUS_CONNECTION_STRING";
```

Comment:
This rename was inadvertent, as I removed the `const` at one point and in restoring used the pattern from my original copy.   I can restore if it offends anyone.